lib/manageiq/active_record_connector.rb

```diff
@@ -0,0 +1,58 @@
+require "active_record"
+unless defined? DatabaseConfigurationPatch
```

Comment:
it does...require/require_relative etc are all safe from double require, regardless of how you pass the path (they normalize to fully qualified under the covers)